src/api/resource-environment-variables.js

```diff
@@ -0,0 +1,12 @@
+export const makeUrl = (appName, envName, componentName) => {
```

Comment:
The exports in this file is only imported and used through redux/sagas, but to me it doesn't look like envvar state is used from redux. envvars are read using usePollEnvVars.

Reply:
File has been deleted now.